### ISSUE TITLE
Menu tree publish / unpublish

### ISSUE THREAD

**Issue by retailify** [divio/django-cms#5154]
djangoCMS Release: 3.2.3
Given following site structure:
```
- parent 1: published
+--> child 1: published
+--> child 2: published
```
If I copy child 1 or child 2 into the same parent 1 as child 3, all childs of parent 1 are unpublished. The same problme exists if I drag and drop another unpublished page to parent 1!

```
- parent 1: published
+--> child 1: unpublished
+--> child 2: unpublished
+--> child 3: unpublished
```
If I unpublish and publish parent 1 again:
```
- parent 1: published
+--> child 1: published
+--> child 2: published
+--> child 3: unpublished
```

**Comment by czpython** [comment]
@retailify Thanks for reporting.
These issues have been fixed on the 3.2.x branch.
Do you mind using that branch for your tests?

Related tickets:
https://github.com/divio/django-cms/issues/5107
https://github.com/divio/django-cms/issues/5140

**Comment by czpython** [comment]
Fixed on develop and backported to 3.2.x
